assets/js/lib/analytics/index.test.js

```diff
@@ -9,4 +11,14 @@ describe('analytics config', () => {
 
     expect(getFromConfig('analyticsEnabled')).toBeTruthy();
   });
+
+  it('it should allow analytics opt-in to be configurable', () => {
+    expect(posthog.has_opted_out_capturing()).toBeFalsy();
```

Comment:
Why is false at the beginning? The opt out is the default thing, right?

Reply:
I just added it in the beginning because I was wanted to confirm the initial value. I have removed this line since it is tested at the end anyway.